### PR TITLE
Add encoding/decoding times to console

### PR DESCRIPTION
**Pull request by RReverser**
This intentionally excludes time of loading corresponding modules, and only measures actual processing.

While this is not perfect as it's not integrated in the UI (cc @jakearchibald), it gives at least some way to measure performance of different codecs and their integrations on particular files.

### CODE REVIEW

File: src/codecs/processor-worker/index.ts [hunk 1]
```diff
@@ -2,13 +2,18 @@ import { expose } from 'comlink';
...
+}
+
 async function mozjpegEncode(
   data: ImageData, options: import('../mozjpeg/encoder-meta').EncodeOptions,
 ): Promise<ArrayBuffer> {
   const { encode } = await import(
     /* webpackChunkName: "process-mozjpeg-enc" */
     '../mozjpeg/encoder');
-  return encode(data, options);
+  return timed('mozjpegEncode', () => encode(data as any, options));
```
Comment (RReverser) [inline src/codecs/processor-worker/index.ts:16]: This one was an accident, reverting.

Comment (RReverser) [reply src/codecs/processor-worker/index.ts]: Done.

**Comment by RReverser** [comment]
@surma Was thinking about it, but AFAIK for those you need to explicitly start Performance measurement in DevTools, which would also instrument and measure all the other functions? If so, seems a bit like overkill if so.

**Comment by RReverser** [comment]
In the long run, I'd like this to somehow be part of UI anyway, but @jakearchibald said it might be complicated as we don't have much free space left.

For now I'll merge with at least console timings, but open to any improvements in future.

**Comment by nucliweb** [comment]
What points does it show on a layer, like in debug mode?

**Comment by nucliweb** [comment]
What points does it show on a layer, like in debug mode?

**Pull request merged**